### ISSUE TITLE
Write a plugin for Thunderbird

### ISSUE THREAD

**Issue by KeizerDev** [Vogeltak/magister-email#1]
Haha, would be cool to see that! :smiley:

**Comment by Vogeltak** [comment]
Yeah that would be cool. Although it wouldn't have much use. Teachers and students can't use Thunderbird on school computers. Besides, this is just a project for me to use Flask more.

**Comment by tkon99** [comment]
We have an issue open on our private simplyHomework repo. It will soon:
- forward emails "github style" (just reply to our email to send it to Magister)
- integrate with CalDAV (calendar sync for Magister appointments)

Although "soon" is pretty relative :wink:

**Comment by Vogeltak** [comment]
Sounds cool! Especially integration with CalDAV sounds very appealing to me, that would sure really useful! Be sure to let me know of your advancements and if I can help out with something.
